server/src/server/session/JanusGraphFactory.java

```diff
@@ -61,25 +55,9 @@
 final public class JanusGraphFactory {
     private final static Logger LOG = LoggerFactory.getLogger(JanusGraphFactory.class);
     private static final AtomicBoolean strategiesApplied = new AtomicBoolean(false);
-    private static final String JANUS_PREFIX = "janusmr.ioformat.conf.";
-    private static final String STORAGE_BACKEND = "storage.backend";
-    private static final String STORAGE_HOSTNAME = ConfigKey.STORAGE_HOSTNAME.name();
     private static final String STORAGE_KEYSPACE = ConfigKey.STORAGE_KEYSPACE.name();
-    private static final String STORAGE_BATCH_LOADING = ConfigKey.STORAGE_BATCH_LOADING.name();
-    private static final String STORAGE_REPLICATION_FACTOR = ConfigKey.STORAGE_REPLICATION_FACTOR.name();
-
-    //These properties are loaded in by default and can optionally be overwritten
-    private static final Properties DEFAULT_PROPERTIES;
-
-    static {
-        String DEFAULT_CONFIG = "resources/default-configs.properties";
-        DEFAULT_PROPERTIES = new Properties();
-        try (InputStream in = JanusGraphFactory.class.getClassLoader().getResourceAsStream(DEFAULT_CONFIG)) {
-            DEFAULT_PROPERTIES.load(in);
-        } catch (IOException e) {
-            throw new RuntimeException(ErrorMessage.INVALID_PATH_TO_CONFIG.getMessage(DEFAULT_CONFIG), e);
-        }
-    }
+    private static final String STORAGE_BACKEND = ConfigKey.STORAGE_BACKEND.name();
+    private static final String CQL_BACKEND = "cql";
```

Comment:
Moved `storage.backend` configuration back inside the code as it is not a configuration that can be actually changed at run-time, as everything would break